serviceUsage.html

```diff
@@ -345,8 +362,94 @@ <h2>
                 <td>When specified message protocol is not loaded</td>
             </tr>
         </table>
-
-    </section>
+		<h2>
+            <a id="examples" class="anchor" href="#examples" aria-hidden="true">
+                <span aria-hidden="true" class="octicon octicon-link"></span>
+            </a>
+        </h2>
+		<h2> Lookups </h2>
+		<p> Presently, to generate Eiffel messages, Eiffel REMREM-Generate is being used. This process requires input mapping exactly one-to-one to Eiffel message Schema.</p>
+		<p> The "Link" field of Eiffel Message requires UUIDs of earlier sent events to link to. But these UUIDs are not always available. Passing these UUID's each time is a little bit difficult, some times user lookup the ER and manually place those IDs in REMReM Generate semi-structured event. In order to resolve this , we need to provide LOOKUP's for the Link's field ,which will have eventType and Properties. eventType will contain type of event.</p>
```

Comment:
I think this text is wrong, if the UUIDs are not available then the lookups also will fail.
The main intention to introduce lookups were to ease the users of mandatorily adding the event Ids that they should know. Introducing this concept will ease them in knowing just what event they  want to query with the parameters so that they are automatically fetched and linked by Generate.